packaging/python/Dockerfile

```diff
@@ -11,24 +11,19 @@ RUN yum -y install \
     unzip \
     bison \
     flex \
-    pkg-config \
     autoconf \
     automake \
-    make \
     openssh-server \
-    libtool
+    libtool \
+    cmake3
 
 WORKDIR /root
 
-RUN curl -L -o cmake-3.12.3.tar.gz https://github.com/Kitware/CMake/releases/download/v3.12.3/cmake-3.12.3.tar.gz \
-    && tar -xvzf cmake-3.12.3.tar.gz \
-    && cd cmake-3.12.3 \
-    && ./bootstrap -- -DCMAKE_INSTALL_PREFIX=/nrnwheel/cmake \
-    && make -j install
+RUN ln -sf /usr/bin/cmake3 /usr/bin/cmake
```

Comment:
Interesting... no binary called only `cmake` ?

Reply:
There is one, but it's version 2.x  - say hello to CentOS :) 
This is why we force `cmake` to  yum installed `cmake3`

Reply:
👍